controllers/drplacementcontrol_controller_test.go

```diff
@@ -691,32 +690,28 @@ func deleteNamespaceMWsFromAllClusters(namespace string) {
 }
 
 func setDRPCSpecExpectationTo(action rmn.DRAction, preferredCluster, failoverCluster string) {
-	localRetries := 0
-	for localRetries < updateRetries {
-		latestDRPC := getLatestDRPC()
+	drpcLookupKey := types.NamespacedName{
```

Comment:
Since this is used only in the context of this small helper, and we look up and update one resource, we can use `key` instead of `drpclookupkey`.

This also works for other helpers you modified in this commit.

Reply:
Considering this a nit hence merged